src/parameterized_tendencies/radiation/radiation_utilities.jl

```diff
@@ -30,7 +14,7 @@ Loads data from an `NCDataset` from the `RRTMGP.jl` artifact stored in
 dataset.
 """
 function rrtmgp_data_loader(fn, file_name)
-    NC.Dataset(rrtmgp_artifact(file_name), "r") do ds
+    NC.Dataset(file_name, "r") do ds
         fn(ds)
     end
 end
```

Comment:
Do we still need this function?

Reply:
The function should be replaced with what was introduced here: https://github.com/CliMA/RRTMGP.jl/pull/520

Reply:
(Which you did)